Project21/Project21/conversionfile.c

```diff
@@ -0,0 +1,36 @@
+// ------------------------------------------------------------------------------------------------
+// Assignment 3 ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.Naveenkumar s
+// ------------------------------------------------------------------
+// Assignment 3
+// Decimal to Binary conversion and Decimal to HexaDecimal conversion
+// ------------------------------------------------------------------------------------------------
+#define INT_SIZE sizeof(int)*8
+#define HEX_SIZE 8

```

Comment:
Also, the name int_size is not indicative of what the variable actually does. It is returning the number of bits you need to store the converted value. The name of the variable should reflect that